common/v2/features/PageNotFound/PageNotFound.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import './PageNotFound.scss';
```

Comment:
I simply copied the component from v1 and wanted to avoid refactoring components
I created a card to refactor this one: https://app.clubhouse.io/mycrypto/story/2433/fix-pagenotfound